tests/test_octodns_manager.py

```diff
@@ -309,6 +336,19 @@ def populate(self, lenient=False):
             with self.assertRaises(TypeError):
                 manager._populate_and_plan('unit.tests.', [NoZone()], [])
 
+    def test_alias_zones(self):
+        with TemporaryDirectory() as tmpdir:
+            environ['YAML_TMP_DIR'] = tmpdir.dirname
+
+            Manager(get_config_filename('simple-alias-zone.yaml')) \
+                .validate_configs()
+
+            with self.assertRaises(ManagerException) as ctx:
+                Manager(get_config_filename('unknown-source-zone.yaml')) \
+                    .validate_configs()
```

Comment:
I don't understand why this test does not cover this lines: https://github.com/github/octodns/pull/592/files#diff-1ed4adeb738f3f2202e89f6a229378fd0bcaf8467a61f694c34b73e24169280eR465

Reply:
I believe it does run through the block as you expect. The lack of coverage is for the other case, where the zone does exist. What the coverage is saying is that things ALWAYS go into that block and never skip it and make it down to the `continue`.